src/workers/ans104-data-indexer.ts

```diff
@@ -83,12 +92,24 @@ export class Ans104DataIndexer {
         typeof item.data_size === 'number'
       ) {
         log.debug('Indexing data item data...');
+        await this.contiguousDataIndex.saveDataContentAttributes({
+          id: item.id,
+          hash: item.data_hash,
+          dataSize: item.data_size,
+          contentType: item.content_type,
+          cachedAt: currentUnixTimestamp(),
```

Comment:
Let's drop `cachedAt` here. We shouldn't include it unless we're actually writing the data to the cache. I may have been the one who said we needed it too, my bad is so.

Reply:
[`b20b953` (#102)](https://github.com/ar-io/ar-io-node/pull/102/commits/b20b953ce83da4395881be89d71b14eae724d09c)